pom.xml

```diff
@@ -118,6 +118,21 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.3.2</version>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.10</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
```

Comment:
I am trying to make it work with java util logging, but we need to add an system property: -Djava.util.logging.config.file=logging.properties and I am not sure where should I add it programmatically. any thoughts?